cmd/nodecmd/validateSubnet.go

```diff
@@ -189,78 +194,68 @@ func validateSubnet(_ *cobra.Command, args []string) error {
 	if blockchainID == ids.Empty {
 		return ErrNoBlockchainID
 	}
-	ansibleNodeIDs, err := ansible.GetAnsibleHostsFromInventory(app.GetAnsibleInventoryDirPath(clusterName))
-	if err != nil {
-		return err
-	}
-	nodeIDMap, failedNodesMap := getNodeIDs(ansibleNodeIDs)
+	nodeIDMap, failedNodesMap := getNodeIDs(hosts)
 	failedNodes := []string{}
```

Comment:
replace `failedNodes := []string{} nodeErrors := []error{}` with `map[string]error` no need to have 2 arrays when you can have one map

Reply:
This don't believe for current PR, but will do it